hpcm/hpe-mlds-build-hpcm-os-image.md

```diff
@@ -377,18 +400,25 @@ master_host: $(/opt/clmgr/bin/cm node show -n master -M --display-no-header | aw
 master_port: 8080
 resource_pool: compute-pool
 EOF
+chmod 640 /etc/determined/agent.yaml
 echo "### PRINT AGENT CONFIG START"
 cat /etc/determined/agent.yaml
 echo "### PRINT AGENT CONFIG END"
 
 # Run the Agent
-docker run -d --restart unless-stopped --name hpe-mlde-agent --network host --gpus all -v /var/run/docker.sock:/var/run/docker.sock -v /etc/determined/agent.yaml:/etc/determined/agent.yaml determinedai/hpe-mlde-agent:0.17.15
+docker run -d --restart unless-stopped --name hpe-mlde-agent \
+  --log-driver=journald \
+  --network host \
+  --gpus all \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v /etc/determined/agent.yaml:/etc/determined/agent.yaml \
+  determinedai/hpe-mlde-agent:0.17.15
 ```
 
 And run:
 
 ```bash
-chmod 700 /root/hpe-mlde-agent-cuda-start.sh
+chmod 750 /root/hpe-mlde-agent-cuda-start.sh
```

Comment:
It's usually safe to mark files executable files under `/root` as `750` since they're owned by the `root` group by default.